chargeio/interface/IPayment.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace chargeio
+{
+    public interface IPayment
```

Comment:
What is this interface? From the single method, it looks like dummy/placeholder code, in which case it should be removed.